packages/pds/src/sequencer/events.ts

```diff
@@ -1,74 +1,61 @@
-import { CID } from 'multiformats/cid'
 import { z } from 'zod'
-import { cborEncode, schema } from '@atproto/common'
-import {
-  BlockMap,
-  CidSet,
-  CommitData,
-  WriteOpAction,
-  blocksToCarFile,
-} from '@atproto/repo'
+import { cborEncode, noUndefinedVals, schema } from '@atproto/common'
+import { BlockMap, blocksToCarFile } from '@atproto/repo'
 import { AccountStatus } from '../account-manager'
-import { PreparedWrite } from '../repo'
+import { CommitDataWithOps } from '../repo'
 import { RepoSeqInsert } from './db'
 
 export const formatSeqCommit = async (
   did: string,
-  commitData: CommitData,
-  writes: PreparedWrite[],
+  commitData: CommitDataWithOps,
 ): Promise<RepoSeqInsert> => {
-  let tooBig: boolean
-  const ops: CommitEvtOp[] = []
-  const blobs = new CidSet()
-  let carSlice: Uint8Array
-
   const blocksToSend = new BlockMap()
   blocksToSend.addMap(commitData.newBlocks)
   blocksToSend.addMap(commitData.relevantBlocks)
 
-  // max 200 ops or 1MB of data
-  if (writes.length > 200 || blocksToSend.byteSize > 1000000) {
-    tooBig = true
+  let evt: CommitEvt
+
+  // If event is too big (max 200 ops or 1MB of data)
+  if (commitData.ops.length > 200 || blocksToSend.byteSize > 1000000) {
```

Comment:
Just a note, think we're still anticipating the byte size check and how it gets handled to be updated.